CharlieBackend.Api.UnitTes/DashboardServiceTest.cs

```diff
@@ -357,81 +607,137 @@ public async Task GetStudentResult()
                     }
                 });
 
-            dashbordRepositoryMock.Setup(x => x.GetStudentAverageVisitsPercentageByStudentIdsAsync(studentIdWithoutGroup, new List<long>()))
+            _unitOfWorkMock.Setup(x => x.DashboardRepository).Returns(_dashboardRepositoryMock.Object);
+            var currentUserServiceAsStudentWithGroup = GetCurrentUserAsExistingStudent(entityId: studentIdWithGroup);
+
+            var dashbordServiceWithGroup = new DashboardService(_unitOfWorkMock.Object, currentUserServiceAsStudentWithGroup.Object);
+
+            //Act
+            var resultWithData = await dashbordServiceWithGroup.GetStudentResultAsync(studentIdWithGroup, dashbordAnaliticRequstWithData);
+
+            //Assert
+            resultWithData.Data.AverageStudentsMarks.Should().NotBeNull();
```

Comment:
Please verify the expected result value of EverageStudetsMarks

Reply:
Fixed